UltimateAFK/AFKComponent.cs

```diff
@@ -102,6 +102,8 @@ private void AFKChecker()
 
             if (plugin.Config.TryReplace && !this.IsPastReplaceTime())
             {
+                var role = Exiled.Loader.Plugins.FirstOrDefault(pl => pl.Name == "EasyEvents")?.Assembly.GetType("EasyEvents.Util")?.GetMethod("GetRole")?.Invoke(null, new object[] {this.ply});
```

Comment:
I'm pretty sure this gets overwritten by the variable on line 122, is it possible to change this to a different variable name?